pkg/apis/pipeline/v1alpha1/task_defaults.go

```diff
@@ -16,12 +16,25 @@ limitations under the License.
 
 package v1alpha1
 
-import "context"
+import (
+	"context"
+	"fmt"
+)
 
 func (t *Task) SetDefaults(ctx context.Context) {
 	t.Spec.SetDefaults(ctx)
 }
 
+// SetDefaults set any defaults for the task spec
 func (ts *TaskSpec) SetDefaults(ctx context.Context) {
+	if ts.Outputs != nil && len(ts.Outputs.Resources) > 0 {
+		for i, o := range ts.Outputs.Resources {
+			if o.Type == PipelineResourceTypeImage {
+				if o.OutputImagePath == "" {
+					ts.Outputs.Resources[i].OutputImagePath = fmt.Sprintf("/tools/image-outputs/%s", o.Name)
```

Comment:
hm `/tools/image-outputs` seems different from the default in the docs? (`/workspace/<resource name>/`?)

Reply:
the default value of `tools/image-outputs` is used if no value is provided

in the docs it says
```
If no value is specified for `outputImagePath`, it will default to `/tools/image-outputs/{resource-name}`.
```

I used a `workspace/<resource-name>` in the example to show how to provide this value